setup.py

```diff
@@ -41,6 +41,7 @@ def find_version(*parts):
             'django-cadmin = configurations.management:execute_from_command_line',
         ],
     },
+    install_requires=['six'],
```

Comment:
`six` could be removed from `extras_require/testing`.

Reply:
Sure thing

Reply:
Good catch